### ISSUE TITLE
Animated GIF should be simpler

### ISSUE THREAD

**Issue by ZevEisenberg** [Raizlabs/RaisinToast#10]
The image of an actual piece of toast is using up a lot of colors in the GIF's indexed color palette, meaning that the rest of it needs to use dithering to express the colors. If the toast were replaced with a solid color (for the GIF only), the GIF would look better and probably have a smaller file size.

**Comment by adamhrz** [comment]
@ZevEisenberg Tried a different header image but still got dithering. Could it be the ffmpeg settings?
![raisin-toast-6plus2](https://cloud.githubusercontent.com/assets/8091449/5846365/09092f64-a191-11e4-831d-00d09b514bc9.gif)

**Comment by ZevEisenberg** [comment]
Yes. Some time let me take a crack at the original video file in Photoshop. It gives you really good control over GIF export settings. ffmpeg might as well, but in Photoshop you get real-time, interactive feedback.
